setup.py

```diff
@@ -44,8 +44,7 @@
     },
     classifiers=[
```

Comment:
Adding `python_requires` helps pip install the right version:

```suggestion
    python_requires='>=3.6',
    classifiers=[
```

Reply:
I did not know about this, thanks!